test/ServerRequestTest.php

```diff
@@ -97,7 +97,19 @@ public function testRemovingAttributeReturnsCloneWithoutAttribute($request)
         $this->assertNull($new->getAttribute('foo', null));
     }
 
-    public function testUsesProvidedConstructorArguments()
+    public function provideMethods()
+    {
+        return array(
```

Comment:
Agreed; that's one thing I've enjoyed about developing on phly/http and now diactoros. :)